src/openrct2-ui/windows/ViewClipping.cpp

```diff
@@ -68,10 +68,11 @@ class ViewClippingWindow final : public Window
 {
 private:
     CoordsXY _selectionStart;
-    CoordsXY _previousClipSelectionA;
-    CoordsXY _previousClipSelectionB;
+    CoordsXY _previousClipSelectionA = { 0, 0 };
+    CoordsXY _previousClipSelectionB = { MAXIMUM_TILE_START_XY, MAXIMUM_TILE_START_XY };
```

Comment:
I still think these changes do not belong to this PR. It's a different issue, not directly related to giant screenshots.

Reply:
Reverted the file entirely to its original state. Once this PR is finished, that one is the next.